PSGSuite/Public/Drive/Get-GSDriveFileList.ps1

```diff
@@ -113,7 +122,6 @@ function Get-GSDriveFileList {
         try {
             $request = $service.Files.List()
             $request.SupportsTeamDrives = $true
-            $request.Fields = 'files,kind,nextPageToken'
```

Comment:
@WJurecki - Previously, `'files,kind,nextPageToken'` was what was always being sent for the Fields parameter with no option to specify the Fields desired due to missing the parameter on the function itself. To prevent an disruption, I set the default value of the newly added `Fields` parameter to `@('files','kind','nextPageToken')`, resolving to the same. 

To your point though in #177 , since it now only gets attached to the body if the Fields parameter is explicitly used, the default value never is passed so Fields isn't set by default anymore.